src/main/java/one/util/streamex/EntryStream.java

```diff
@@ -689,6 +689,12 @@
      * @param keyPredicate a non-interfering, stateless predicate to apply to
      *        the key of each element to determine if it should be included
      * @return the new stream
+     * @see #nonNull()
+     * @see #filter(Predicate)
+     * @see #remove(Predicate)
+     * @see #selectKeys(Class)
+     * @see #selectValues(Class)
+     * @see EntryStream  EntryStream specific version with "Keys|Values|KeyValue" suffix

```

Comment:
Why linking the same class from the class itself?

Reply:
I'm not sure I get your point on this.

If your question is regarding the purpose of all these `@see`, it's just because I (as a user) don't know your cool API by heart. Hence I tend to rewrite some boilerplate code you already have in your methods. Adding these `@see` I hope to prevent other StreamEx users to write `.filter(Objects::nonNull)` and discovering `.nonNull()` far too late 

If your question is regarding the fact that in `EntryStream` my  `@see` include the current method, I can correct it if you want, this was just a faster way to "link" each filtering method to one another with just a big no-brainer Copy/Paste

Reply:
My question was only about `EntryStream` link. It looks redundant indeed. Ok, I'll fix redundant links on my side.  Thanks for the contribution.